extensions/mongodb/deployment/src/main/java/org/apache/camel/quarkus/component/mongodb/deployment/MongoDbProcessor.java

```diff
@@ -28,4 +30,9 @@
     FeatureBuildItem feature() {
         return new FeatureBuildItem(FEATURE);
     }
+
+    @BuildStep
+    ReflectiveClassBuildItem reflectiveClass() {
+        return new ReflectiveClassBuildItem(true, false, ChangeStreamDocument.class);
```

Comment:
If you create an issue on the Quarkus side, please create one in this project cross referencing it so that we remember to remove this.

Reply:
Yes I mentioned it in them main comment of this PR. This is exactly what I want to do.